composer.json

```diff
@@ -8,7 +8,8 @@
         "spryker/kernel": "^3.30.0",
         "spryker/log": "^3.0.0",
         "spryker/monolog": "^2.0.0",
-        "spryker/symfony": "^3.5.0"
+        "spryker/symfony": "^3.5.0",
+        "composer/semver": "^3.0.0"
```

Comment:
Why it was added as a dependency?

Reply:
🤔 
removed

Reply:
The dependency is still there, please verify